docs/src/pages/components/no-ssr/FrameDeferring.js

```diff
@@ -12,7 +12,7 @@ const useStyles = makeStyles({
 });
 
 function LargeTree() {
-  return Array.from(new Array(3000)).map((_, index) => <span key={index}>.</span>);
+  return [...Array(3000)].map((_, index) => <span key={index}>.</span>);
```

Comment:
Hmm, I don't seem to get an error on IE11

Reply:
We probably polyfill this accidentally. I'd say it's safer to move away from Array.from in case the polyfill gets removed in an unrelated change.

https://kangax.github.io/compat-table/es6/

Reply:
:+1:

Reply:
No idea why this doesn't crash. If I go into the IE11 dev console `Array.from` is `undefined`. But in the demo it somehow works.

Reply:
So Array.from is required anyway for babel to work: https://babeljs.io/docs/en/caveats#polyfills

Is babel doing this now automatically?